web/public/locales/sv.json

```
@@ -98,27 +157,29 @@
     "dataIsDelayed": "Data är fördröjd"
   },
   "misc": {
-    "maintitle": "Aktuellt CO₂-utsläpp från elproduktion",
+    "maintitle": "Aktuellt 24/7 CO₂-utsläpp från elproduktion",
     "oops": "Aj då! Vi har problem att nå servern. Vi kommer att försöka igen om några sekunder.",
     "newversion": "En ny version är tillgänglig! Tryck <a onClick=\"location.reload(true);\">här</a> för att ladda om.",
     "webgl-not-supported": "Kartan kan inte renderas då denna webbläsare inte stöder WebGL.",
     "retrynow": "Försök igen",
     "database-ad": "Letar du efter historisk data? <a href=\"https://electricitymap.org?utm_source=app.electricitymap.org&utm_medium=referral\" target=\"_blank\">Kolla in vår databas!</a>",
     "api-ad": "Vill ha uppdaterad data i din app eller på din enhet? <a href=\"https://electricitymap.org?utm_source=app.electricitymap.org&utm_medium=referral\" target=\"_blank\">Prova vårt API!</a>",
     "legend": "Förklaring",
-    "faq": "Vanliga frågor"
+    "faq": "Vanliga frågor",
+    "privacyPolicy": "Integritetspolicy",
+    "legalNotice": "Rättsligt meddelande"
   },
   "wind": "vind",
-  "windDataError": "Vind data är för nuvarande otillgängligt",
+  "windDataError": "Vinddata är för närvarande otillgänglig",
   "solar": "sol",
-  "solarDataError": "Sol data är för nuvarande otillgängligt",
+  "solarDataError": "Soldata är för närvarande otillgänglig",
   "hydro": "vatten",
   "hydro storage": "pumpkraft",
   "battery storage": "batterilager",
   "biomass": "biomassa",
   "nuclear": "kärnkraft",
   "geothermal": "geotermisk",
-  "gas": "naturgas",
+  "gas": "gas",
```

```suggestion
  "gas": "naturgas",
```
I think it makes sense to keep it as natural gas so it's not confused with biogas or hydrogen in the future.

---

They have changed from "natural gas" to just "gas" in en.json. So I guess we should reflect that change.